pyseq.py

```diff
@@ -876,6 +1060,40 @@ def get_sequences(source):
             seq = Sequence([item])
             seqs.append(seq)
 
-    log.debug('time: %s' % (datetime.now() - start))
-
-    return list(seqs)
+    log.debug("time: %s" %(datetime.now() - start))
+    if not stereo:
+        log.info("added sequences: %s" %(seqs))
+        return seqs
+    else:
+        seqs.sort()
```

Comment:
isnt that what I am doing, up in _find_view_pairs ?
if the string before the view matches and the string after the view matches then it is a pair....
@rsgalloway 
sure using a modified diff is much nicer